site/class-2-style/zezavdh/index.js

```diff
@@ -0,0 +1,23 @@
+function render(b,c){
```

Comment:
This isn’t the prettiest style 😢 (subjectively!) You didn’t rename your one letter variables, most stuff is on one line... But fine, I’ll merge :)